GPflow/priors.py

```diff
@@ -88,6 +104,10 @@ def __init__(self, mu, sigma):
     def logp(self, x):
         return tf.reduce_sum(densities.laplace(self.mu, self.sigma, x))
 
+    def sample(self, shape=()):
+        return np.array(np.random.laplace(self.mu, self.sigma,
+                                          size=shape if shape else None))
```

Comment:
I believe (1,) gives you a 1-d vector with one element which is ever so slightly different from a numpy scalar (e.g. you can index the latter).
I wanted scalars to be default, but I also wanted everything to be Numpy for consistency.